data/crac/crac-util/src/main/java/com/farao_community/farao/data/crac_util/CracAliasesUtil.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright (c) 2020, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_util;
+
+import com.farao_community.farao.data.crac_api.*;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.*;
+
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static com.farao_community.farao.data.crac_util.UcteNodeMatchingRule.*;
+
+/**
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+
+public final class CracAliasesUtil {
+
+    static final int NB_CHARACTER_IN_UCTE_NODE = 9;
```

Comment:
not 8 ? :dancers:

Reply:
I rename this element: it represents the beginning of the second node (8 + one blank).